packages/design-system/src/atoms/tooltipV2/index.tsx

```diff
@@ -5,6 +5,7 @@ import { TypographyV2 } from '../typographyV2'
 export interface TooltipV2Props
   extends Omit<TooltipProps, 'title' | 'variant'> {
   title: string
+  visible?: boolean
```

Comment:
What's the reason for this change?

Reply:
That change is now removed as it's redundant. It was added when I was trying out another solution.